### PR TITLE
FEA: Avatar Clothing

### PR DESCRIPTION
**Pull request by kolexxx**
This let players use their terry avatars.

### CODE REVIEW

File: code/Game.Convars.cs [hunk 1]
```diff
@@ -92,4 +92,7 @@ public static void UpdateVoiceChat( bool oldValue, bool newValue )
 		}
 	}
 	#endregion
+
+	[ConVar.Server( "ttt_avatar_clothing" )]
+	public static bool AvatarClothing { get; set; } = true;
```
Comment (mzegar) [inline code/Game.Convars.cs:97]: should be `false`

File: code/Game.cs [hunk 2]
```diff
@@ -5,134 +5,149 @@ namespace TTT;
...
+        Detective.Hat = ResourceLibrary.Get<Clothing>( "models/detective_hat/detective_hat.clothing" );
+
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/hat/balaclava/balaclava.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/hair/eyebrows/eyebrows_black.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/jacket/longsleeve/longsleeve.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/gloves/tactical_gloves/tactical_gloves.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/trousers/smarttrousers/trousers.smart.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/vest/tactical_vest/models/tactical_vest.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/shoes/trainers/trainers.clothing" ) );
+    }
```
Comment (mzegar) [inline code/Game.cs:145]: Put this method in `Player.Clothes.cs` imo

File: code/Game.Convars.cs (hunk 1 above)
Comment (kolexxx) [reply code/Game.Convars.cs:97]: It will be `true` for now

File: code/Game.cs (hunk 2 above)
Comment (kolexxx) [reply code/Game.cs:145]: I think this is fine.

**Pull request merged**